### ISSUE TITLE
DeviceSupport 12.2 (16E227)

### ISSUE THREAD

**Issue by ThomasVolkert** [iGhibli/iOS-DeviceSupport#40]
Hi there,
I am searching for DeviceSupport for version 12.2 (16E227).
BR.

**Comment by evisoup** [comment]
16E226 works the same

**Comment by ThomasVolkert** [comment]
Yes, this is my workaround for the moment. But I hoped there are some official files out there.

**Comment by Sunrise17** [comment]
I have tried both iOS 12.2(16E226) and 12.2 (16E5212e) on my iPad Air 2 device from xCode 10.1. Unfortunately, it causes memory leak issue and when i have checked "Leaks" from instrument, it shows +8000 leaks. Will (16E5227a) solve this issue? When i try app on iPhone 6 having iOS
12.1.4 version, it works without an issue (0 leaks).

**Comment by wang93wei** [comment]
https://www.lanzous.com/i3psbbi

12.2（16E227）

**Comment by Sunrise17** [comment]
@wang93wei , thanks for your sharing, i have tried with your file but unfortunately issue still goes on. I have asked it to [StackOverFlow](https://stackoverflow.com/questions/55587856/memory-leak-occurs-in-ipad-air-2-device). In iPad Air 2 simulator (iOS 12.1), in iPhone 6 (iOS 12.1) devices, no memory leak issue, i have seen in only iPad Air 2 real device (iOS 12.2)
![issue](https://user-images.githubusercontent.com/26038249/55792554-92d6fc80-5ac9-11e9-86d3-a02a6f32e38e.png)

**Comment by chrishanyu** [comment]
@Sunrise17 try restart xcode and reconnect your device

**Comment by Sunrise17** [comment]
@chrishanyu , i have tried many times, even i clear the ram of iPad Air 2. However, still no many leaks seen as above in iPad Air 2. In contrast, in iPhone 6(12.1.4) and all other simulators (12.1 version), i have not seen any leak.

**Comment by LuisBonsembiante** [comment]
anyone could help me...I not have a solution for that

**Comment by NaiyerAghaz** [comment]
I was also facing with this problem couples of days now get a complete solutions and is working fine for 12.2 (16E226), if not working after added the only 12.2 (16E226) then you should add also these file 12.2 (16E5181e),12.2 (16E5212e).
I hope work fine now.
